### DISCUSSION TITLE
dcm compression params

### DISCUSSION THREAD

**Discussion by linzhidong1984** [dcm4che/dcm4chee-arc-light#3304] [Q&A] [answered]
dcm4chee-arc-light 5.23.1

i use j2ki(1.2.840.10008.1.2.4.91) to compress image from 35M to 3.7M. But 3.7M is still bigger than i expect.
so how can i  compress image smaller than 3.7M, 2M is the best. like config some compression params.
thanks for your help!

**Comment by gunterze** [comment, marked as answer]
You may adjust **Image Write Param**: `compressionRatiofactor=10.0` of [Image Writer](https://dcm4chee-arc-cs.readthedocs.io/en/latest/networking/config/imageWriter.html) for **Transfer Syntax**: `1.2.840.10008.1.2.4.91` to your needs.
![Screenshot from 2021-08-10 12-19-03](https://user-images.githubusercontent.com/400095/128850452-5242c3f3-9a33-4c4b-bbbd-86c5196f99f1.png)
